.gitignore

```
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# environment variables
+./env
```

```suggestion
.env
```

the slash here will assume a directory called `env` but it is a single file in the root folder